src/components/page-intro.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatLargeNumber } from '../utils';
+import '../styles/components/page-intro.scss';
+import ChevronDown from '../svg/chevron-down.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+
+class PageIntro extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showContent: false,
+    };
+  }
+
+  render() {
+    const {
+      title, resultsCount, children, links,
+    } = this.props;
+    const { showContent } = this.state;
+    return (
+      <div className="page-intro">
+        <h2>
+          <button
+            type="button"
+            onClick={() => this.setState({ showContent: !showContent })}
+            className="dropdown"
+          >
+            {showContent ? <ChevronUp /> : <ChevronDown />}
+            {' '}
+            {title}
+          </button>
+          {resultsCount > 0 && (
+          <small>
+            {' '}
+            {formatLargeNumber(resultsCount)}
+            {' '}
+results
+          </small>
+          )}
+        </h2>
+
+        <div className="intro-content" style={{ display: showContent ? 'block' : 'none' }}>
+          {children}
+          <div className="intro-links">
+            {links.map(link => (
```

Comment:
I don't know the coding convention we follow here, but instead of `link` we can have `({ title, destination })`. If we follow this everywhere, the help on more complicated components will be more pronounced.

Reply:
I think in general it's nicer to use destructuring, but in this case `title` is already destructured as another variable in the upper scope. I could rename it while destructuring but I think that makes things a bit more confusing...

Reply:
Okay, fair enough.